include/fs/fs_interface.h

```diff
@@ -31,11 +31,17 @@ extern "C" {
 #endif /* filesystem selection */
 #endif /* CONFIG_FILE_SYSTEM_MAX_FILE_NAME */
 
+
 /* Type for fs_open flags */
 typedef uint8_t fs_mode_t;
 
 struct fs_mount_t;
 
+/**
+ * @ingroup file_system_api
```

Comment:
shouldn't @addtogropu be used instead?

Reply:
I think that `@ingroup` will work the same way as `@addgroup`.

Reply:
no, it only include the subsequent item.
So `struct fs_dir_t` is still not documented (I checked by build the doc locally).

Reply:
I have failed to notice. Expected it to do the same thing. Will fix.